src/components/AccountDrawer/index.tsx

```diff
@@ -24,6 +27,8 @@ export const AccountCard: React.FC<{ account: Account }> = ({ account }) => {
 
   const { openWith } = useContext(DynamicModalContext);
 
+  account = getImplicitAccountSafe(account.address.pkh)!;
+
```

Comment:
If you are using the `!` operator maybe it makes sense to use the get account unsafe instead?

Also, looks like it does not make sense anymore to pass in the account, but it's better to use the PKH instead because you are going to fetch it anyway

And with that in mind it might be still useful to use the selected account context

Reply:
I checked other places, yeah, it's fine to leave it as is